pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[tool.poetry]
+name = "envattrs"
+version = "19.1"
+description = "Populate attr.s or dataclass objects from environment variables"
+authors = ["Jonas Obrist <jonas.obrist@hennge.com>"]
+license = "APLv2"
+repository = "https://github.com/HENNGE/envattrs"
+readme = "README.md"
+homepage = "https://github.com/HENNGE/envattrs"
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.6",
+  "Programming Language :: Python :: 3.7",
+]
```

Comment:
Are newer Pythons not supported?

Reply:
> > Preserve current support of Python 3.6
> 
> we can drop it: https://endoflife.date/python

Updated to python 3.7

Reply:
Updated to include until 3.11. Tested on 3.7 and 3.11. Will try to add automated testing for all python versions.